### PR TITLE
[CORE-2036] Add singular default and defaults global help

### PR DESCRIPTION
**Pull request by robert-uhl**

### CODE REVIEW

File: src/server/pps/cmds/cmds.go [hunk 1]
```diff
@@ -1659,7 +1665,7 @@ func Cmds(mainCtx context.Context, pachCtx *config.Context, pachctlCfg *pachctl.
 	}
 	inspectDefaults.Flags().BoolVar(&cluster, "cluster", false, "Inspect cluster defaults.")
 	inspectDefaults.Flags().StringVar(&project, "project", project, "Inspect project defaults.")
-	commands = append(commands, cmdutil.CreateAliases(inspectDefaults, "inspect defaults"))
+	commands = append(commands, cmdutil.CreateAliases(inspectDefaults, "inspect defaults", "default"))
```
Comment (acohen4) [inline src/server/pps/cmds/cmds.go:1668]: nit: in other sites where we use alias synonyms. We have the singular form in the command string and plurals in the synonyms list. Could you modify to that for consistency?

File: src/server/pps/cmds/cmds.go [hunk 2]
```diff
@@ -1625,6 +1625,12 @@ func Cmds(mainCtx context.Context, pachCtx *config.Context, pachctlCfg *pachctl.
 	rerunPipeline.Flags().StringVar(&project, "project", project, "Specify the project (by name) containing project")
 	commands = append(commands, cmdutil.CreateAlias(rerunPipeline, "rerun pipeline"))
 
+	defaultsDocs := &cobra.Command{
+		Short: "Docs for defaults.",
+		Long:  `Defaults provide default values for certain requests, e.g. when creating a pipeline.`,
+	}
+	commands = append(commands, cmdutil.CreateDocsAliases(defaultsDocs, "defaults", " defaults", "default"))
```
Comment (acohen4) [inline src/server/pps/cmds/cmds.go:1632]: same here: swap second and fourth arguments

**Review by acohen4 — APPROVED** [review]
One nit, otherwise LGTM

File: src/server/pps/cmds/cmds.go (hunk 1 above)
Comment (robert-uhl) [reply src/server/pps/cmds/cmds.go:1668]: I had considered that, but defaults are always referred to as plurals, which is different from the other nouns. For example, `pachctl inspect repo` is the normal way of inspecting a repo (but `pachctl inspect repos` is allowed), but `pachctl create defaults --cluster` is the normal way of inspecting cluster defaults (but `pachctl inspect default --cluster` is allowed). @maguenza, do you have any preference here?

Comment (robert-uhl) [reply src/server/pps/cmds/cmds.go:1668]: The way it is now:

```
$ pachctl help default
Defaults provide default values for certain requests, e.g. when creating a pipeline.

Associated Commands:
  pachctl create defaults   Set cluster defaults.
  pachctl delete defaults   Delete defaults.
  pachctl inspect defaults  Return defaults.
  pachctl update defaults   Update defaults.
```

With the swap:

```
$ pachctl help default
Defaults provide default values for certain requests, e.g. when creating a pipeline.

Associated Commands:
  pachctl create default   Set cluster defaults.
  pachctl delete default   Delete defaults.
  pachctl inspect default  Return defaults.
  pachctl update default   Update defaults.
```

Comment (maguenza) [reply src/server/pps/cmds/cmds.go:1668]: @acohen4 @robert-uhl - after thinking about it and discussing with customer eng, we think it makes sense to keep it as `defaults` and the alias be `default`. The reason being it's grammatically correct and more intuitive for users. I think doing that way would be a better UX.

**Pull request merged**